File: fizzbuzz.pl
#!/usr/bin/perl
# vim: ts=2

use strict;
use warnings;

for ($a=1;$a<=100;$a++)
{
	if ($a % 15 == 0)
	{
		print "FizzBuzz\n";
	}
	elsif ($a % 3 == 0)
	{
		print "Fizz\n";
	}
	elsif ($a % 5 == 0)
	{
		print "Buzz\n";
	}
	else
	{
		print "$a\n";
	}
}

File: fizzbuzz.rb
#!/usr/bin/env ruby

for elem in (1..100)
  if elem % 15 == 0
    puts "FizzBuzz"
  elsif elem % 5 == 0
    puts "Buzz"
  elsif elem % 3 == 0
    puts "Fizz"
  else
    puts elem
  end
end
